packages/bpk-component-image/src/BpkImage.module.scss

```diff
@@ -37,6 +37,6 @@
   }
 
   &--border-radius-sm {
-    @include radii.bpk-border-radius-sm;
+    @include radii.bpk-border-radius-md;
   }
```

Comment:
Hi, is it better to keep the sytle name consistent with the size? so that user can better to understand the actual radius size they are using.